src/Android/Utilities/AndroidHelpers.cs

```diff
@@ -47,5 +49,22 @@ public static async Task SetPreconfiguredRestrictionSettingsAsync(Context contex
                 await AppHelpers.SetPreconfiguredSettingsAsync(dict);
             }
         }
+
+        public static PendingIntentFlags PendingIntentMutability(PendingIntentFlags pendingIntentFlags, bool isMutable)
+        {
+            //Mutable flag was added on API level 31
+            if ((isMutable && Build.VERSION.SdkInt > BuildVersionCodes.R) || Build.VERSION.SdkInt < BuildVersionCodes.M)
```

Comment:
Shouldn't the condition be just `if (isMutable && Build.VERSION.SdkInt > BuildVersionCodes.R)`?
why did you added `|| Build.VERSION.SdkInt < BuildVersionCodes.M`?

Reply:
You're right. That comparison was on a earlier version, tried to simplify the if and probably this was added back on an undo.